### PR TITLE
Added Contour

### PR DESCRIPTION
**Pull request by flymia**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

This adds the [contour](https://github.com/contour-terminal/contour) terminal emulator as a Cask.

### CODE REVIEW

File: Casks/contour.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  url "https://github.com/contour-terminal/contour/releases/download/v#{version}/contour-#{version}-osx.dmg"
+  name "Contour"
+  desc "Modern terminal emulator, for everyday use"
+  homepage "https://github.com/contour-terminal/contour/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (SMillerDev) [inline Casks/contour.rb:13]: Do they use pre-releases?

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/contour.rb (hunk 1 above)
Comment (flymia) [reply Casks/contour.rb:13]: > Do they use pre-releases?

For previous releases they did use pre-releases, yes. I don't know if there will be any more.

**Comment by flymia** [comment]
> Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

I this necessary, even if the app itself does not create any config files, etc.? The only artifact that is included is `/Applications/Contour.app` - should I add this to the zap?

A config file will be created by the user, as the command reference of contour says.

**Comment by miccal** [comment]
> The only artifact that is included is `/Applications/Contour.app` - should I add this to the zap?

No.

> A config file will be created by the user, as the command reference of contour says.

Please add this to the `zap` stanza, thanks.

> I checked the zaps using AppCleaner and nothing more gets created.

Thanks.

**Comment by miccal** [comment]
Thank you @flymia.

**Review by miccal — APPROVED** [review]

**Pull request merged**